Marsiple/Comment/CommentViewController.swift

```diff
@@ -41,7 +41,8 @@ private extension CommentViewController {
                                                         cancelActionHandler: { [weak self] _ in
                                                             self?.navigationController?.popViewController(animated: true) })
                                         self?.present(alert, animated: true, completion: nil) },
-                                      failure: { [weak self] message in
+                                      failure: { [weak self] error in
+                                        let message = error.errorDescription ?? ""
```

Comment:
We should use the `localizedDescription` property here. It seems a bit confusing, but `localizedDescription` uses the `errorDescription` property of our custom `LocalizedError` conforming types. 

Therefore, we should use `localizedDescription` when extracting the localized message.